docs/diagnostics/PX1054.md

```diff
@@ -3,32 +3,86 @@ This document describes the PX1054 diagnostic.
 
 ## Summary
 
-| Code   | Short Description                                                                               | Type  | Code Fix    | 
-| ------ | ----------------------------------------------------------------------------------------------- | ----- | ----------- | 
-| PX1054 | A `PXGraph` instance cannot start a long-running operation during the `PXGraph` initialization. | Error | Unavailable |
+| Code   | Short Description                                                                                     | Type  | Code Fix    | 
+| ------ | ----------------------------------------------------------------------------------------------------- | ----- | ----------- | 
+| PX1054 | Long-running operations cannot be started during the `PXGraph` and `PXGraphExtension` initialization. | Error | Unavailable |
```

Comment:
@EAndrosova please review

Reply:
Should we explain why the long running operations cannot be started during PXGraph initialization?